README.md

```diff
@@ -23,6 +23,17 @@
 > Base branch for unomi changes is `rfk-unomi-v2.1`.  
 > Please merge all the changes to `rfk-unomi-v2.1` branch only 
 
+Deployment Steps
+============
+- Create a feature branch. Make changes raise PR.
+- Merge PR with `rfk-unomi-v2.1` branch.
+- Switch to `rfk-unomi-v2.1` branch and run `mvn package` or `mvn package -DskipTests`
+- This build the jar under `package/target` directory
+- Take backup of the jar present in s3 path `s3://rfk-repo/packages/rfk-data-unomi/` to `s3://rfk-repo/packages/rfk-data-unomi/unomi-2.0.0_bkp`
+- Copy the jar `unomi-2.0.0.tar.gz` to s3 path `s3://rfk-repo/packages/rfk-data-unomi/`.
+
+Note: This jar will be pulled by `rfk-data-unomi` repo [here](https://github.com/Reflektion/rfk-data-unomi/blob/main/Dockerfile#L36) while creating unomi image.
```

Comment:
To be specific, these are not jar files. They are binary. Please replace `jar` with `binary` everywhere. 
```suggestion
Note: This binary will be pulled by `rfk-data-unomi` repo [here](https://github.com/Reflektion/rfk-data-unomi/blob/main/Dockerfile#L36) while creating unomi image.
```